WORKSPACE

```diff
@@ -59,7 +59,7 @@ maven_install(
         "io.opentracing:opentracing-mock:0.33.0",
         "commons-collections:commons-collections:3.2.2",
         "org.awaitility:awaitility:4.1.0",
-        "commons-cli:commons-cli:1.4",
+        "commons-cli:commons-cli:1.5.0",
```

Comment:
Should the version in pom be upgraded as well?

Reply:
pom has upgraded!